PurchasesCoreSwift/Misc/DictionaryExtensions.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  DictionaryExtensions.swift
+//  PurchasesCoreSwift
+//
+//  Created by César de la Vega on 7/21/21.
+//  Copyright © 2021 Purchases. All rights reserved.
+//
+
+import Foundation
+
+extension Dictionary {
+
+    func removingNSNullValues() -> Dictionary {
+        self.filter { !($0.value is NSNull) }
+    }
```

Comment:
we already have this: 
`purchases-ios/PurchasesCoreSwift/FoundationExtensions/Dictionary+Extensions.swift`

That version has the `rc_` prefix. It's not strictly necessary, though, since it's an internal extension. But we should unify them

Reply:
Oh, interesting. I will unify them